### PR TITLE
Deploy on Azure

### PR DESCRIPTION
**Pull request by florian-vuillemot**

### CODE REVIEW

File: docs/manuals/run-deploy/deploy/azure.md [hunk 1]
```diff
@@ -1,4 +1,102 @@
...
+- The `--logs` flag configures default logging required to enable viewing the log stream immediately after launching the web app.
+
+You can optionally specify the `--name <app-name>` argument to provide a custom name for your app. If not provided, a name will be automatically generated.
+
+The command may take a few minutes to complete. While it is running, it provides messages about resource group creation, App Service plan creation, app resource creation, logging configuration, and ZIP deployment. Once completed, it will display the message "You can launch the app at http://<app-name>.azurewebsites.net", which is the URL of your app.
+
+!!! Note
+
+    The command can also include the `--location <location-name>` argument to specify the Azure region. Use `az account list-locations` to retrieve a list of available regions for your account.
+
```
Comment (jrobinAV) [inline docs/manuals/run-deploy/deploy/azure.md:23]: I believe we should have an example with a screenshot of the running app here.

File: docs/manuals/run-deploy/deploy/azure.md [hunk 2]
```diff
@@ -1,4 +1,102 @@
...
+Login to Azure using the command:
+```
+az login
+```
+
+Create the web app and other necessary resources, then deploy your code to Azure using the following command:
+```
+az webapp up --runtime PYTHON:3.9 --sku B1 --logs
+```
```
Comment (jrobinAV) [inline docs/manuals/run-deploy/deploy/azure.md]: I don't get how we push the code on remote.

**Review by jrobinAV — COMMENTED** [review]
Great PR. It s not a draft to me.

File: docs/manuals/run-deploy/deploy/azure.md (hunk 1 above)
Comment (florian-vuillemot) [reply docs/manuals/run-deploy/deploy/azure.md:23]: I'm going to delete this part. It's too azure-oriented.

File: docs/manuals/run-deploy/deploy/azure.md [hunk 3]
```diff
@@ -1,4 +1,83 @@
 # Deploy on Azure
 
-TODO: [taipy-doc#397](https://github.com/Avaiga/taipy-doc/issues/397)
+[Azure] (https://azure.microsoft.com/en-us), one of the world's leading providers of cloud services, is the perfect platform for deploying Taipy applications. It allows developers to stay focused on their application bringing effortless scalability, monitoring and security natively. The following documentation shows how to deploy on Azure App Service and get an application into the hands of the end-user.
```
Comment (FabienLelaquais) [inline docs/manuals/run-deploy/deploy/azure.md]: "the perfect platform" is maybe a little bit too strong?
"a very relevant choice"?

File: docs/manuals/run-deploy/deploy/azure.md [hunk 4]
```diff
@@ -1,4 +1,83 @@
...
 
+## Prerequisites
+
+- Minimal knowledge of Azure.
+- Azure CLI should be installed. Check [the official documentation](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) for azure CLI installation.
```
Comment (FabienLelaquais) [inline docs/manuals/run-deploy/deploy/azure.md]: azure -> Azure

File: docs/manuals/run-deploy/deploy/azure.md [hunk 5]
```diff
@@ -1,4 +1,83 @@
...
+- [:material-arrow-right: Running a Taipy application](../run/index.md)
+
+
+## Azure App Service
+
+[Azure App Service](https://learn.microsoft.com/en-us/azure/app-service/) is a managed platform that simplifies web application deployment, scaling, and management. Azure Web App, a service within App Service, specializes in hosting and managing web applications. As a Python developer, you can leverage Azure Web App to quickly deploy and run your Python-based web applications in a scalable, hassle-free environment.
+
+!!! Note
+
+    You can create Azure App Service and Azure Web App using Azure portal, VS Code, Azure Tools extension pack, or Azure CLI. Here we will focus on Azure CLI.
```
Comment (FabienLelaquais) [inline docs/manuals/run-deploy/deploy/azure.md]: portal -> Portal

Comment (FabienLelaquais) [inline docs/manuals/run-deploy/deploy/azure.md]: "Here we will" -> "In this section, we"

File: docs/manuals/run-deploy/deploy/azure.md [hunk 6]
```diff
@@ -1,4 +1,83 @@
...
+
+## Prepare your application
+
+Open a terminal and go in the application folder. Ensure you have a `requirements.txt` file listing the required dependencies and your entry point Python file named `app.py` or `application.py`. These files are crucial for Azure App Service to deploy your application correctly.
```
Comment (FabienLelaquais) [inline docs/manuals/run-deploy/deploy/azure.md]: "go in" -> "set your directory to"

File: docs/manuals/run-deploy/deploy/azure.md [hunk 7]
```diff
@@ -1,4 +1,83 @@
...
+```
+
+## Create a Web App in Azure
+
+In your terminal, log in to Azure using the command:
+```shell
+taipy@taipy:~$ az login
+```
+
+Still in your terminal, you can create the web app and then deploy your code using the following command:
```
Comment (FabienLelaquais) [inline docs/manuals/run-deploy/deploy/azure.md]: "Still in your terminal, you can create the web app"
->
"You can now create the web application"

File: docs/manuals/run-deploy/deploy/azure.md [hunk 8]
```diff
@@ -1,4 +1,83 @@
...
+Configuring default logging for the app, if not already enabled
+Creating zip with contents of dir /home/taipy ...
+Getting scm site credentials for zip deployment
+Starting zip deployment. This operation can take a while to complete ...
+Deployment endpoint responded with status code 202
+You can launch the app at http://gentle-stone-7d284754337a4dcb968d392baac1ccc9.azurewebsites.net
+Configuring default logging for the app, if not already enabled
+```
+
+- The `--runtime` parameter specifies the Python version your app is running, in this case, Python 3.9.
```
Comment (FabienLelaquais) [inline docs/manuals/run-deploy/deploy/azure.md]: "app" -> "application"

File: docs/manuals/run-deploy/deploy/azure.md [hunk 9]
```diff
@@ -1,4 +1,83 @@
...
+- The `--sku` parameter defines the size (CPU, memory) and cost of the app service plan. This example uses the B1 (Basic) service plan.
+- The `--logs` flag configures the default logging required to enable viewing the log stream immediately after launching the web app.
```
Comment (FabienLelaquais) [inline docs/manuals/run-deploy/deploy/azure.md]: app -> application

File: docs/manuals/run-deploy/deploy/azure.md [hunk 10]
```diff
@@ -1,4 +1,83 @@
...
+
+You can optionally specify the `--name <app-name>` argument to provide a custom name for your app. If not provided, a name will be automatically generated.
```
Comment (FabienLelaquais) [inline docs/manuals/run-deploy/deploy/azure.md]: app -> application

"If not" -> "If it is not"

File: docs/manuals/run-deploy/deploy/azure.md [hunk 11]
```diff
@@ -1,4 +1,83 @@
...
+
+The command may take a few minutes to complete. While running, it provides messages about the resource group and the Azure App Service names. Keep them for the next step. In the example, the resource group name is `taipy_rg_4939` and the App Service is `taipy_asp_8825`.
+
+Once completed, it will display the message "You can launch the app at http://<app-name>.azurewebsites.net", which is the URL of your app. In the example `http://gentle-stone-7d284754337a4dcb968d392baac1ccc9.azurewebsites.net`.
```
Comment (FabienLelaquais) [inline docs/manuals/run-deploy/deploy/azure.md]: app -> application

**Review by FabienLelaquais — APPROVED** [review]
Minor typographic changes.

**Pull request merged**